tests/model_selection/test_random_split.py

```diff
@@ -0,0 +1,180 @@
+        assert fold_info["Train"] == train_users.size
+        assert fold_info["Train users"] == pd.unique(train_users).size
+        assert fold_info["Train items"] == pd.unique(train_items).size
+        assert fold_info["Test"] == test_users.size
+        assert fold_info["Test users"] == pd.unique(test_users).size
+        assert fold_info["Test items"] == pd.unique(test_items).size
+
+    @pytest.mark.parametrize("execution_number", range(5))
+    def test_cold_users(self, interactions: Interactions, train_size: float, execution_number: list) -> None:
+        interactions_copy = deepcopy(interactions)
+        rs = RandomSplitter(train_size, True, False, False)
+
+        actual = list(rs.split(interactions, collect_fold_stats=True))
+        pd.testing.assert_frame_equal(interactions.df, interactions_copy.df)
+
+        df = interactions.df
+        train_users = pd.unique(df.iloc[actual[0][0]]["user_id"])
+        test_users = pd.unique(df.iloc[actual[0][1]]["user_id"])
+
+        assert np.intersect1d(train_users, test_users).shape[0] == test_users.shape[0]
+
+    @pytest.mark.parametrize("execution_number", range(5))
+    def test_cold_items(self, interactions: Interactions, train_size: float, execution_number: list) -> None:
+        interactions_copy = deepcopy(interactions)
+        rs = RandomSplitter(train_size, False, True, False)
+
+        actual = list(rs.split(interactions, collect_fold_stats=True))
+        pd.testing.assert_frame_equal(interactions.df, interactions_copy.df)
+
+        df = interactions.df
+        train_items = pd.unique(df.iloc[actual[0][0]]["item_id"])
+        test_items = pd.unique(df.iloc[actual[0][1]]["item_id"])
+
+        assert np.intersect1d(train_items, test_items).shape[0] == test_items.shape[0]
+
+    @pytest.mark.parametrize("execution_number", range(5))
+    def test_filter_already_seen(self, interactions: Interactions, train_size: float, execution_number: list) -> None:
+        interactions_copy = deepcopy(interactions)
+        rs = RandomSplitter(train_size, False, False, True)
+
+        actual = list(rs.split(interactions, collect_fold_stats=True))
+        pd.testing.assert_frame_equal(interactions.df, interactions_copy.df)
+
+        df = interactions.df
+        train_interactions = df.iloc[actual[0][0]][["user_id", "item_id"]]
+        test_interactions = df.iloc[actual[0][1]][["user_id", "item_id"]]
+
+        assert train_interactions.merge(test_interactions, how="inner").shape[0] == 0
+
+    @pytest.mark.parametrize(
+        "incorrect_train_size, expected_error_type",
+        (
+            (-0.1, ValueError),
+            (1.5, ValueError),
+        ),
+    )
+    def test_with_incorrect_train_size(
+        self, interactions: Interactions, incorrect_train_size: float, expected_error_type: tp.Type[Exception]
+    ) -> None:
+        with pytest.raises(expected_error_type):
+            RandomSplitter(incorrect_train_size, False, False, False)
+
+
+class TestGetNotSeenMask:
```

Comment:
As you don't need a second version of function, you also don't need a second version of test